src/app/components/pages/AdminUserManager.tsx

```diff
@@ -110,18 +235,41 @@ export const AdminUserManager = () => {
   const modifyUserEmailVerificationStatusesAndUpdateResults = async (status: EmailVerificationStatus) => {
     setUserUpdating(true);
     const selectedEmails =
-      searchResults?.filter((user) => user.id && selectedUserIds.includes(user.id)).map((user) => user.email || "") ||
+      searchResults?.filter((user) => user.id && selectedUserIds.includes(user.id)).map((user) => user.email ?? "") ??
       [];
     await dispatch(adminModifyUserEmailVerificationStatuses(status, selectedEmails));
     dispatch(adminUserSearchRequest(searchQuery));
     setSelectedUserIds([]);
     setUserUpdating(false);
   };
 
-  const search = (event: React.FormEvent<HTMLFormElement>) => {
-    event.preventDefault();
-    setSearchRequested(true);
+  const declineTeacherUpgradeAndUpdateResults = async () => {
+    setUserUpdating(true);
+    const response = await dispatch(adminModifyTeacherPending(false, selectedUserIds));
     dispatch(adminUserSearchRequest(searchQuery));
+    setSelectedUserIds([]);
+    setUserUpdating(false);
+    if (response) dispatch(showSuccessToast("Teacher Pending Status", response));
+  };
+
+  const confirmUnverifiedUserPromotions = function () {
+    if (searchResults.length) {
```

Comment:
There isn't anything wrong with the way you've implemented this, but given that you're not checking a boolean (`lenght` is a `number`), you could have also simply said `if (searchResults) {`. In both cases you're testing whether something is truthy or falsy.